operations/restrictions.md

```diff
@@ -369,3 +369,111 @@ Removes restrictions from the service.
 ```javascript
 {}
 ```
+
+## Set restrictions
+
+Adds new restrictions with the specified conditions.
+
+If there already exists a restriction with the same conditions, following scenarios apply:
+
+1) If the exceptions of the new restriction match the old restriction:
+   1) If the new interval is longer than the old one, new restriction is created joining the two intervals.
+   2) If the new interval is shorter, no changes are made.
+2) If the exceptions of the new restriction do not match the old restriction:
+   1) If the new interval is overlaps the old interval, the old restriction will be spliced before and after the new interval. Restrictions matching the old restriction are then added at the appropriate interval along with the new restriction.
+   2) If the new interval do not overlap the old interval, the new restriction is added as usual.
+
+If the supplied restrictions match in all the properties but differ in interval and follow each other chronologically, the supplied restrictions will be joined into a single restriction.
+
+The usage of this endpoint must be enabled per enterprise.
```

Comment:
How, where, who can do it?

Reply:
So who can do it for them? Should I write something like "contact the technical partner support team"?